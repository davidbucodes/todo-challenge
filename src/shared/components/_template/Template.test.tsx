import { composeStories } from "@storybook/react";
import { render, screen } from "@testing-library/react";
import * as stories from "./Template.stories";

const { Default } = composeStories(stories);

describe("Template", () => {
  describe("Default", () => {
    it("should render correctly", () => {
      const { container } = render(<Default />);
      const wrapperElement = screen.getByText(/Template/i);
      expect(container).toMatchSnapshot();
      expect(wrapperElement).toBeInTheDocument();
    });
  });
});
