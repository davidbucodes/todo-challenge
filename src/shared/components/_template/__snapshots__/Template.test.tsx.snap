// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Template Default should render correctly 1`] = `
<div>
  <div
    class="sc-bgqQcB"
  >
    Template
  </div>
</div>
`;
